doc/7/controllers/index/stats/snippets/stats.js

```diff
@@ -0,0 +1,3 @@
+const stats = await kuzzle.index.stats();
+
+console.log(JSON.stringify(stats));
```

Comment:
Can you add the JSON structure in comment please?

So user know how to manipulate the result then